Support for receiving plain java objects

petternordholm opened the pull request:
We are using this excellent bundle in one of our projects. Our code is connecting to a legacy system using JMS on activemq and need to be able to receive ObjectMessages.

This PR adds support for receiving serialized java objects without doing any json deserialization.

petternordholm committed:
Support for receiving plain java objects (#1)

This commit adds support for receiving serialized java objects
without doing any json deserialization.

petternordholm committed:
Add trusted packages

A new configuration option has been added enabling the developers
to specify packages safe to receive on the consumer side.

mbknor commented:
Hi, And thanks for both the interest in the project AND this PR.
I've looked at it and it looks good, but can you please update the README, so that others can also take advantage of this improvement.

Please also explain what the trustedPackage-thing is.

When you have updated the docs, I'll merge it and create a new release

petternordholm commented:
Great!

I'll updated the PR with an updated README.

petternordholm committed:
Object messages in README.md

README.md has been updated with information on how to receive
object messages. Configuration related to trusted packages
is also described.

mbknor commented:
thanks a lot..

mbknor commented:
0.5.2 is now released and should be on its way to the maven central. thanks again
